templates/pulp_workers

```diff
@@ -0,0 +1,8 @@
+# Configuration file for Pulp's Celery workers
+
+# Define the number of worker nodes you wish to have here. This defaults to the number of processors
+# that are detected on the system if left commented here.
+PULP_CONCURRENCY=<%= @num_workers %>
+
+# Configure Python's encoding for writing all logs, stdout and stderr
+PYTHONIOENCODING="UTF-8"
```

Comment:
I believe the contents of this file depend on el6 vs el7:

https://github.com/pulp/pulp/blob/master/server/etc/default/systemd_pulp_workers
https://github.com/pulp/pulp/blob/master/server/etc/default/upstart_pulp_workers

On my el6 box, i have the other file.